### PR TITLE
Implement CSV file parsing

### PR DESCRIPTION
**Pull request by pavlokhomiak**

### CODE REVIEW

**Review by dmytro-hryhoruk — COMMENTED** [review]
travis failed..
 Rather it's a problem in travis.yml replace it with this version

language: java
jdk:
  - openjdk11
script:
  - mvn clean install
cache:
  directories:
    - $HOME/.m2
before_install:
  - chmod +x mvnw

File: src/main/java/mate/academy/boot/hellobootdemo/util/CsvFileParser.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+public class CsvFileParser {
+
+    public List<ReviewDto> parseDataToDto(List<String> dataList) {
+        return IntStream.range(1, dataList.size())
+                .mapToObj(i -> parseToDto(dataList.get(i)))
+                .collect(Collectors.toList());
+    }
+
+    private ReviewDto parseToDto(String data) {
+        String[] dataArray = data.split(",");
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/boot/hellobootdemo/util/CsvFileParser.java:17]: does it work properly on all file objects? what if some text field contains comma?

File: src/test/java/mate/academy/boot/hellobootdemo/util/CsvFileParserTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            + "HelpfulnessNumerator,HelpfulnessDenominator,Score,Time,Summary,Text";
+    private static final String EXPECTED_FIRST_ROW = "38,B001EO5QW8,A3MGP2E1ZZ6GRB,\"Zardoz \"\"focuspuller\"\"\""
+            + ",1,1,5,1238457600,it's oatmeal,\"What else do you need to know? Oatmeal, instant (make it with a "
+            + "half cup of low-fat milk and add raisins\tnuke for 90 seconds). More expensive than something. It's "
+            + "still just oatmeal. Mmm, convenient!\"";
+    private static final String EXPECTED_SECOND_ROW = "39,B001EO5QW8,A2GHZ2UTV2B0CD,JERRY REITH,0,0,4,1350777600,"
+            + "GOOD WAY TO START THE DAY....,\"I WAS COFFEE , HE CAME WITH ( A PACKET OF McCANNS INSTANT IRISH OATMEAL "
+            + ".) HE SUGGESTED THAT I TRY IT FOR MY OWN USE ,IN MY STASH . A CHANCE TO SAY NO , APPLE AND CINN .\"";
+    private static final List<String> INPUT_DATA = List.of(EXPECTED_HEADER, EXPECTED_FIRST_ROW, EXPECTED_SECOND_ROW);
+    private static final List<ReviewDto> EXPECTED_LIST = List.of(
```
Comment (dmytro-hryhoruk) [inline src/test/java/mate/academy/boot/hellobootdemo/util/CsvFileParserTest.java:22]: only if you can't remove or add something to collection(immutable collection) and it's consists only of immutable objects you can call it constant

Comment (dmytro-hryhoruk) [inline src/test/java/mate/academy/boot/hellobootdemo/util/CsvFileParserTest.java:22]:
```suggestion
    private static final List<ReviewDto> expectedReviews = List.of(
```

File: src/test/java/mate/academy/boot/hellobootdemo/util/CsvFileParserTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    private static final String EXPECTED_HEADER = "Id,ProductId,UserId,ProfileName,"
+            + "HelpfulnessNumerator,HelpfulnessDenominator,Score,Time,Summary,Text";
+    private static final String EXPECTED_FIRST_ROW = "38,B001EO5QW8,A3MGP2E1ZZ6GRB,\"Zardoz \"\"focuspuller\"\"\""
+            + ",1,1,5,1238457600,it's oatmeal,\"What else do you need to know? Oatmeal, instant (make it with a "
+            + "half cup of low-fat milk and add raisins\tnuke for 90 seconds). More expensive than something. It's "
+            + "still just oatmeal. Mmm, convenient!\"";
+    private static final String EXPECTED_SECOND_ROW = "39,B001EO5QW8,A2GHZ2UTV2B0CD,JERRY REITH,0,0,4,1350777600,"
+            + "GOOD WAY TO START THE DAY....,\"I WAS COFFEE , HE CAME WITH ( A PACKET OF McCANNS INSTANT IRISH OATMEAL "
+            + ".) HE SUGGESTED THAT I TRY IT FOR MY OWN USE ,IN MY STASH . A CHANCE TO SAY NO , APPLE AND CINN .\"";
+    private static final List<String> INPUT_DATA = List.of(EXPECTED_HEADER, EXPECTED_FIRST_ROW, EXPECTED_SECOND_ROW);
```
Comment (dmytro-hryhoruk) [inline src/test/java/mate/academy/boot/hellobootdemo/util/CsvFileParserTest.java:21]: this is constant, cause List.of() returns immutable collection, and it's contain strings

**Review by dmytro-hryhoruk — COMMENTED** [review]
How much does it take to parse the whole file?

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**